docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -751,14 +755,14 @@ public class NativeBearerTokenAuthenticationIT extends BearerTokenAuthentication
 For more information about initializing and configuring Dev Services for Keycloak, see the xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] guide.
 
 ifndef::no-deprecated-test-resource[]
-[[integration-testing-keycloak]]
+[[bearer-token-integration-testing-keycloak]]
```

Comment:
@rolfedh As far as I recall this is the only test option which has been deprecated, but it looks like you are undeprecating it, why ? I know I missed it in the cherry-picked PR...

Reply:
Sorry, you are still keeping it conditionalized